09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,69 @@
+body{
+  font-family:sans-serif;
+}
+main {
+  background-image:url("../img/featured-image-4.jpg");
+  background-repeat: no-repeat;
+  background-position:center;
+  background-size:cover;
+  height:350px;
+  width:100%;
+  text-align: center;
+  padding-top:100px;
+  color:white;
```

Comment:
You are missing the code to darken the background-images, Chris. Look in Slack in the atx-intro channel for the code.